ibridges/sync.py

```diff
@@ -315,7 +315,8 @@ def _copy_local_to_irods(session: Session,
             upload(session=session,
                     local_path=source_path,
                     irods_path=target_path,
-                    overwrite=True)
+                    overwrite=True,
+                    ignore_err=True)
```

Comment:
Why are errors ignore by default here? The safe thing to do is throwing the error I think.

Reply:
Not in the sync, in my opinion. Just imagine your are synchronising a large folder and just because of one corrupted file all fails. Unllike the upload and download, by default I think the sync is used interactively. If you use it in remote computations then you need to set the flag differently.

Reply:
I understand the inconvenience that can be associated with it. But to me it's a matter of "how bad":

ignore_err==False -> User needs to manually copy things over/ fix issues with admin -> annoying
ignore_err==True -> User doesn't see message thinks all his files are safely stored on the irods server -> loses data -> disaster

I prefer a larger amount of annoyed users over a possible failure with data loss personally.

Reply:
To account for all possible use cases, I introduced `ignore_err` as a parameter in `sync_data`